eksupgrade/src/latest_ami.py

```diff
@@ -14,15 +14,15 @@ def get_latest_ami(cluster_version: str, instance_type: str, image_to_search: st
     client = boto3.client("ec2", region_name=region)
 
     if "Amazon Linux 2" in instance_type:
-        names = [f"/aws/service/eks/optimized-ami/{cluster_version: str}/amazon-linux-2/recommended/image_id"]
+        names = [f"/aws/service/eks/optimized-ami/{cluster_version: s}/amazon-linux-2/recommended/image_id"]
```

Comment:
I believe the type definitions should be removed from here, like:
```suggestion
        names = [f"/aws/service/eks/optimized-ami/{cluster_version}/amazon-linux-2/recommended/image_id"]
```

Reply:
The format specifiers for strings do work with `s` or None. I can remove this for consistency.

Reply:
I will defer to @mbeacom but I don't see the benefit of the type in the formatting since its in the function signature already

Reply:
Let's drop it.

Reply:
Done